### PR TITLE
Fix type issue

### PR DESCRIPTION
**Pull request by bozhi**

### CODE REVIEW

File: airtable/plugin.py [hunk 1]
```diff
@@ -37,7 +37,10 @@ def airtable_url_to_ids(url: str) -> Tuple[AirtableBaseID, AirtableTableID]:
         raise ValueError("Invalid Airtable URL")
 
     base_id = match.group(1)
-    table_id = match.group(2) if match.group(2) else None
+    # When loading the Airtable using the URL that only has the base id (e.g. https://airtable.com/apptpbyiHjjfPecFw)
+    # Airtable automatically drops you into the first table of the base and we don't expect the table_id to be an empty string
+    # if the user copies the URL from the browser.
+    table_id = match.group(2) if match.group(2) else ""
```
Comment (jcharum) [inline airtable/plugin.py:43]: Should this not be an error?  I am confused at what is being fixed here.  If we expect this to be useful for URLs that only have a base ID, should we not handle that gracefully?

Comment (bozhi) [reply airtable/plugin.py:43]: The AirtableTableID has `id` of type `str`: https://github.com/d8e-ai/lutra-plugin/blob/38b5923ffcc4d1d555ac0ce51f589adbf57edda0/airtable/plugin.py#L20

Prior to this change, there was a type issue where we set the `id` to `None` when `id` should have been a string that is not optional.

Comment (jngiam) [reply airtable/plugin.py:43]: If we think the URLs should **always** contain the table id, I'm with Jerry and think this should raise an exception.

Comment (jngiam) [reply airtable/plugin.py:43]: i.e., we only support URLs with both base and table ids for this function

File: airtable/plugin.py [hunk 2]
```diff
@@ -37,7 +37,13 @@ def airtable_url_to_ids(url: str) -> Tuple[AirtableBaseID, AirtableTableID]:
         raise ValueError("Invalid Airtable URL")
 
     base_id = match.group(1)
-    table_id = match.group(2) if match.group(2) else None
+    # When loading the Airtable using the URL that only has the base id (e.g. https://airtable.com/apptpbyiHjjfPecFw)
+    # Airtable automatically drops you into the first table of the base and we don't expect the table_id to be an empty string
+    # if the user copies the URL from the browser.
+    if not match.group(2):
+        raise ValueError("Invalid Airtable table ID")
```
Comment (jngiam) [inline airtable/plugin.py:44]: Can we have a better error message  that says something about the url? This gets propagated to code gen, so useful to have it be a good message

**Pull request merged**